docs/fundamentals/read-operations.txt

```diff
@@ -175,6 +176,124 @@ Use the following syntax to run a find operation that matches all documents:
+``where()`` method. The ``$text`` operator performs a text search on the
+text-indexed fields. The ``$search`` field specifies the text to search for.
+
+A text search retrieves documents that contain a **term** or a **phrase** in the
+text-indexed fields. A term is a sequence of characters that excludes
+whitespace characters. A phrase is a sequence of terms with any number
+of whitespace characters.
+
+.. note::
+
+   Before you can perform a text search, you must create a :manual:`text
+   index </core/indexes/index-types/index-text/>` on
+   the text-valued field. To learn more about creating
+   indexes, see the :ref:`laravel-eloquent-indexes` section of the
+   Schema Builder guide.
+
+After building your query with the ``where()`` method, chain the ``get()``
+method to retrieve the query results.
+
+This example calls the ``where()`` method on the ``Movie`` Eloquent model to
+retrieve documents in which the ``plot`` field contains the phrase
+``"love story"``:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      Use the following syntax to specify the query:
+
+      .. literalinclude:: /includes/fundamentals/read-operations/ReadOperationsTest.php
+         :language: php
+         :dedent:
+         :start-after: start-text
+         :end-before: end-text
+
+   .. tab:: Controller Method
+      :tabid: controller
+
+      To see the query results in the ``browse_movies`` view, edit the ``show()`` function
+      in the ``MovieController.php`` file to resemble the following code:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+
+            class MovieController
+            {
+                public function show()
+                {
+                     $movies = Movie::where('$text', ['$search' => "\"" . 'love story' . "\""])
+                         ->get();
+
+                     return view('browse_movies', [
+                         'movies' => $movies
+                     ]);
+                }
+            }
+
+         .. output::
+            :language: none
+            :visible: false
+
+            Title: Cafè de Flore
+            Year: 2011
+            Runtime: 120
+            IMDB Rating: 7.4
+            IMDB Votes: 9663
+            Plot: A love story between a man and woman ...
+            
+            Title: Paheli
+            Year: 2005
+            Runtime: 140
+            IMDB Rating: 6.7
+            IMDB Votes: 8909
+            Plot: A folk tale - supernatural love story about a ghost ...
+            
+            Title: Por un puèado de besos
+            Year: 2014
+            Runtime: 98
+            IMDB Rating: 6.1
+            IMDB Votes: 223
+            Plot: A girl. A boy. A love story ...
+
+            ...
+            
+.. note::
+
+   To specify a phrase as the text search criteria, you must include it
+   with escaped quotes in the query filter.
```

Comment:
Question:
Perhaps a question for the technical reviewer, but is it possible to use single quotes instead of double quotes to represent the value of the $search key (which would hopefully avoid having to escape the quotes and concatenate strings)?

Reply:
I use [single quote](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.single) everywhere, unless I want to use variable injection with double quotes.

Reply:
tested this improvement and it works! removing the admonition